devops/charts/controller/templates/secrets.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: google-oauth-key
+  labels: {{- include "attestation-controller.labels" . | nindent 4}}
+type: Opaque
+data:
+  google_oauth_key.json: {{.Values.google_oauth_key.json | b64enc}}
```

Comment:
When I used the google auth key locally I don't think I had it base64 encoded. That might break things

Reply:
This won't work locally. Its just for deploying to a kubernets cluster which expects all secrets to be base64 encoded.